config/overlays/odh-xks/README.md

```
@@ -39,23 +39,23 @@ matches the `kserveIngressGateway` setting.
 
 ### TLS Architecture
 
-The overlay uses a pluggable cert-manager architecture with OpenDataHub-scoped CA
-(can be shared by other ODH components):
+The overlay uses a pluggable cert-manager architecture with RHAII-scoped CA
+(can be shared by other RHAII components):
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)